bundles/core/org.eclipse.smarthome.core.thing.test/src/test/java/org/eclipse/smarthome/core/thing/type/ChannelTypeBuilderTest.java

```diff
@@ -30,7 +30,7 @@
 import org.junit.Test;
 
 /**
- * Tests the {@link ChannelTypeBuilder}.
+ * Tests the {@link GenericChannelTypeBuilder}.
```

Comment:
some references to the intermediate `GenericChannelTypeBuilder` which are obsolete now.

Reply:
Good catch. Unfortunately deleting a file is no "refactor" action in Eclipse :(